components/progressTracker/theme.css

```diff
@@ -0,0 +1,180 @@
+@import '@teamleader/ui-colors';
+@import '@teamleader/ui-typography';
+@import '@teamleader/ui-animations';
+
+.tracker {
+  align-items: flex-end;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 20px 24px;
+}
+
+.step {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  max-width: 180px;
+  position: relative;
+
+  &:before {
+    content: '';
+    height: 2px;
+    position: absolute;
+    bottom: 11px;
+  }
+
+  &:after {
+    content: '';
+    height: 4px;
+    position: absolute;
+    bottom: 10px;
+    left: 50%;
+    transition: width var(--animation-duration) var(--animation-curve-default);
+    width: 0;
+    z-index: 1;
+  }
+
+  &:not(:first-child):not(:last-child) {
+    &:before {
+      width: 100%;
+      left: 0;
+    }
+  }
+
+  &:first-child {
+    &:before {
+      width: 50%;
+      left: 50%;
+    }
+  }
+
+  &:last-child {
+    &:before {
+      width: 50%;
+      left: 0;
+    }
+  }
+
+  &.is-completed:not(:last-child) {
+    &:after {
+      width: 100%;
+    }
+  }
+
+  .status-bullet {
+    background-color: transparent;
+    transition: background-color .1s var(--animation-curve-default) calc(var(--animation-duration) - .1s);
+    align-self: center;
+    border-radius: 50%;
+    display: block;
+    height: 12px;
+    margin: 6px 0;
+    width: 12px;
+    z-index: 2;
+    position: relative;
+    &:before {
+      background-color: transparent;
+      transition: background-color .1s var(--animation-curve-default) calc(var(--animation-duration) - .1s);
+      border-radius: 50%;
+      content: '';
+      display: block;
+      position: absolute;
+      width: 6px;
+      height: 6px;
```

Comment:
Nitpicking, but it feels like 12px/6px stuff are just wholes/halves. Could become a variable 🙈

Reply:
that's weird, the precommit hook should catch this